debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/stackwalker/StackWalker.java

```diff
@@ -139,116 +144,148 @@ private static IStackWalker getImpl()
 	private static interface IStackWalker extends IAlgorithm
 	{
 		public StackWalkResult walkStackFrames(WalkState walkState);
-		
+
+		public StackWalkResult walkStackFrames(WalkState walkState, long continuationAddress);
+
 		public void walkBytecodeFrameSlots(WalkState walkState,
 				J9MethodPointer method, UDATA offsetPC,
 				UDATAPointer pendingBase, UDATA pendingStackHeight,
 				UDATAPointer localBase, UDATA numberOfLocals) throws CorruptDataException;
 
-		public StackWalkResult walkStackFrames(WalkState walkState, UDATAPointer sp, UDATAPointer arg0ea, U8Pointer pc, J9MethodPointer literals, J9VMEntryLocalStoragePointer entryLocalStorage);
-
 		FrameCallbackResult walkFrame(WalkState walkState) throws CorruptDataException;
 	}
-	
-	
+
+
 	private static class StackWalker_29_V0 extends BaseAlgorithm implements IStackWalker
 	{
-	
+
 		protected StackWalker_29_V0()
 		{
 			super(90, 0);
 		}
 
-		public StackWalkResult walkStackFrames(WalkState walkState)
-		{
+		@Override
+		public StackWalkResult walkStackFrames(WalkState walkState) {
+			J9VMThreadPointer walkThread = walkState.walkThread;
 			try {
-				return walkStackFrames(walkState, 
-						walkState.walkThread.sp(), 
-						walkState.walkThread.arg0EA(), 
-						walkState.walkThread.pc(), 
-						walkState.walkThread.literals(),
-						walkState.walkThread.entryLocalStorage());
+				walkState.javaVM = walkThread.javaVM();
+				walkState.threadAddress = walkThread.getAddress();
+				walkState.osThread = J9ThreadHelper.getOSThread(walkThread.osThread());
+				walkState.privateFlags = walkThread.privateFlags();
+				if (walkState.arg0EA == null) {
+					walkState.arg0EA = walkThread.arg0EA();
+				}
+				if (walkState.pc == null) {
+					walkState.pc = walkThread.pc();
+				}
+				walkState.pcAddress = walkThread.pcEA();
+				if (walkState.walkSP == null) {
+					walkState.walkSP = walkThread.sp();
+				}
+				if (walkState.literals == null) {
+					walkState.literals = walkThread.literals();
+				}
+				J9VMEntryLocalStoragePointer els = walkThread.entryLocalStorage();
```

Comment:
you should check if an existing ELS is already set on the walkState. (this go together with the user specified ELS parameter)